hasura.planx.uk/migrations/1673514721705_set_fk_public_analytics_flow_id/up.sql

```diff
@@ -0,0 +1,11 @@
+alter table "public"."analytics" drop constraint "analytics_flow_id_fkey",
+  add constraint "analytics_flow_id_fkey"
+  foreign key ("flow_id")
+  references "public"."flows"
+  ("id") on update cascade on delete cascade;
+
+alter table "public"."analytics_logs" drop constraint "analytics_log_analytics_id_fkey",
+  add constraint "analytics_logs_analytics_id_fkey"
+  foreign key ("analytics_id")
+  references "public"."analytics"
+  ("id") on update cascade on delete cascade;
```

Comment:
previously, when an `analytics` record was deleted, it's associated `analytics_log` was configured to set the `analytics_id` field to null, but this triggered another violation because `analytics_id` has a NOT NULL constraint. 

now, we'll simply cascade the flow deletion to _both_ `analytics` & `analytics_logs`.